### ISSUE TITLE
Add options for auto upload and allow manual start of auto upload

### ISSUE THREAD

**Issue by arnowelzel** [nextcloud/android#11448]
### Is your feature request related to a problem? Please describe.
Autoupload needs a regular check for changed files which will need energy. However I do not take pictures all the time and most of the time these checks are completely unneccessary. However I experience more than 200(!) wakelocks in 24 hours which keep the device awake for many hours. Even my e-mail client which is monitoring multiple IMAP IDLE accounts does only trigger about 20-30 wakelocks per day and does not miss any incoming e-mail.

### Describe the solution you'd like
When I've take a picture, the upload does not always need to be done immediately. If it was only once a day or only once an hour this would be fine as well.

It would be nice to have to following options:

1. Set the interval how often the app will check for changes or disable automatic check completely.
2. Have a widget or a button in the app which is easy to reach to manually trigger autouploads if needed.

### Describe alternatives you've considered
Use a dedicated sync app which is only there to synchronize the camera image folder with Nextcloud. However this may not provide the creation of folders for years and months based on the image date and the wakelocks from Nextcloud will still be present since it seems to check for new folders in the background.

### Additional context
(N/A)

**Comment by joshtrichards** [comment]
The *Only upload when charging option* may effectively accomplish what you're looking for (albeit in a different ways than you've requested). 

I use mine that way and have happily for the last 1-2 years. Your concern is why I use this approach. It's under the settings menu for each folder designated for upload.

**Comment by luxzg** [comment]
In a way Josh is right. On the other hand - I'd like a manual "push" button anyway. Just to make sure it started. Maybe it's just me, but I had a history of not uploading with OneDrive, and it continues with NC, app is set fine, but sometimes it just ignores new stuff. Manual sync would maybe help in such cases to nudge it a little

**Comment by TryThisOneMyFriend** [comment]
up!

**Comment by joshtrichards** [comment]
Duplicate of #7019
